tests/integration/test_charm.py

```diff
@@ -44,22 +48,10 @@ async def test_relations(ops_test: OpsTest):
     await ops_test.model.add_relation(f"{ISTIO_PILOT}:ingress", f"{APP_NAME}:ingress")
     await ops_test.model.add_relation(f"{ISTIO_PILOT}:ingress-auth", f"{APP_NAME}:ingress-auth")
 
-    await ops_test.model.wait_for_idle(
-        [APP_NAME, ISTIO_PILOT, DEX_AUTH],
-        status="active",
-        raise_on_blocked=True,
-        raise_on_error=True,
-        timeout=600,
-    )
-
```

Comment:
Istio pilot should be `1.16/stable`

Reply:
let's see if that will work. I did not touch tests as they were

Reply:
cannot bump due  to the bug https://github.com/juju/python-libjuju/issues/851

Reply:
Leaving as is then.